skimage/util/noise.py

```diff
@@ -5,6 +5,36 @@
 __all__ = ['random_noise']
 
 
+def choose(p, shape):
```

Comment:
```suggestion
def _bernoulli(p, shape):
```
I would rather see this function as a private one.